src/components/Good.tsx

```diff
@@ -0,0 +1,12 @@
+type GoodProps = {
+  goodName: string;
+  key: number;
+};
+
+export const Good: React.FC<GoodProps> = ({ goodName, key }) => {
```

Comment:
The `key` prop is not needed in the component's destructuring parameters because it's not a prop that will be passed down to the component. It's used by React internally for list items and should be removed from here.